cloudflare-ddns

```diff
@@ -178,16 +200,20 @@ def main():
     elif args.update_now:
         config = load_configuration()
         if not config:
+            print('There was a problem with the configuration file {config_file}! Try running cloudflare-ddns --configure'.format(config_file=CONFIGURATION_FILE))
             return
-        auth = {'X-Auth-Email': config['email'], 'X-Auth-Key': config['api_key']}
+        if config['auth_type'] == 'token':
+            auth = {'Authorization': 'Bearer {token}'.format(token=config['api_token'])}
+        elif config['auth_type'] == 'key':
+            auth = {'X-Auth-Email': config['email'], 'X-Auth-Key': config['api_key']}
         external_ip = get_external_ip()
         ipv6 = get_ipv6()
-        for domain in config['domains']:
+        for domain in config['domains'].split(','):
             update_dns(domain, auth, external_ip, ipv6)
     else:
         print('No arguments passed; exiting.')
         print('Try cloudflare-ddns --help.')
 
 
 if __name__ == '__main__':
-    main()
+    main()
```

Comment:
Please retain the end-of-file line feed

Reply:
My bad. See 3ea017c6f041d6d2f1a2298babb65a6ad5539aec. Let me know if you need me to do something else . Not sure how to handle this with git.